docs/index.md

```diff
@@ -4,14 +4,23 @@ description: EthSigner is an open-source private transaction manager developed u
 
 # EthSigner Transaction Signer
 
-EthSigner is an open-source, client-agnostic, Ethereum transaction signer developed under the Apache 2.0 license and written in Java. 
+## What is Ethsigner?
 
-EthSigner provides private key signing to separate key management from transaction validation. EthSigner supports [Hashicorp Vault](Using-EthSigner/Hashicorp.md) 
-and [Azure Key Vault](Using-EthSigner/Azure.md). 
+EthSigner is an open-source, client-agnostic, Ethereum transaction signer developed under the Apache 2.0 license and written in Java. It adds a layer of security by signing transactions using a private key that can be secured in a variety of cloud providers, or encrypted on a local disk. EthSigner therefore separates private key management from transaction validation. 
 
-## What can you do with EthSigner? 
+## How does it work?
 
-EthSigner can: 
+EthSigner acts as a proxy service by forwarding requests between the client and network. When it receives a transaction it will generate a signature using the stored private key, and forward the transaction and signature to the client to include in the blockchain.
```

Comment:
This is kind hard - people refer to Pantheon as the "Ethereum Client" - but I'm wondering if in this sentence - its meaning the client _application_?

Reply:
I think Madelines suggestion regarding this paragraph will remove the ambiguity. I'll also be specific and call it an Ethereum client.